Documentation/security/policy/language.rst

```diff
@@ -929,6 +959,8 @@ latter rule will have no effect.
           an *HTTP 403 access denied* is sent back for HTTP requests which
           violate the policy, or a *DNS REFUSED* response for DNS requests.
 
+.. note:: Layer 7 rules do not currently support port ranges.
```

Comment:
Hmm, this section is also getting a bit out-of-control with the `..note::` usage. If there are several notes together, then we are not emphasizing any particular one of them, and that means none of them should be notes.

We can follow up on this separately though, this change is not making things much worse.